web-nfc/NFCReader_options_url-manual.https.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Web NFC Test: NFCReader NFCReaderOptions url test</title>
+<link rel="author" title="Intel" href="http://www.intel.com"/>
+<link rel="help" href="https://w3c.github.io/web-nfc/"/>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<meta name="flags" content="interact">
+
+<p>Tap an NFC tag to the test device with NFC support.</p>
+
+<p>Note: All the actions need to be done in 60 seconds, otherwise it will get TIMEOUT.</p>
+
+<div id="log"></div>
+
+<script>
+
+"use strict";
+
+setup({ explicit_timeout: true });
+
+promise_test(async t => {
+  const writer = new NFCWriter();
+  const reader = new NFCReader({url: `${location.origin}/custom/path`});
+  await writer.push({
+    url: "/custom/path/update",
+    records: [{ recordType: "text", data: "valid" }]
+  });
+  await writer.push({
```

Comment:
This is wrong. You pushed messages twice to an nfc tag, but the latest message stored in the tag is the second one you pushed.

You should read the message every time you pushed and verify the one you are going to filter.